app/views/home/_navbar.html.erb

```diff
@@ -4,6 +4,14 @@
   <ul class="list-group">
     <%= render_vertical_menu :top_menu %>
     <%= render_vertical_menu :admin_menu %>
+    <% if Setting[:experimental_ui] %>
+        <li class="list-group-item secondary-nav-item-pf" id="experimental_ui">
```

Comment:
would this actually work this way? if you are using the react mounter, i would assume we would need to use a content div and not the menu li?